File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.py
def solution(n):
    result=[True]*(n+1)
    result[0]=False
    result[1]=False
    total=[]
    for i in range(2,n+1):
        if result[i]:
            total.append(i)
            for j in range(i*i,n+1,i):
                result[j]=False
    return len(total)

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.js
function solution(n) {
    let result=new Array(n+1).fill(true);
    result[0]=result[1]=false;
    let total=[];
    for(let i=2;i<=n;i++){
        if(result[i]){
            total.push(i);
            for(let j=i*i;j<=n;j+=i){
                result[j]=false;
            }
        }
    }
    return total.length
}